src/Shared/UnitTests/Vacancies.Client/Application/Services/NationalMinimumWageServiceTests.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using Esfa.Recruit.Vacancies.Client.Application.Services.MinimumWage;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Internal;
+using Moq;
+using Xunit;
+
+namespace Esfa.Recruit.Vacancies.Client.UnitTests.Vacancies.Client.Application.Services
+{
+    public class NationalMinimumWageServiceTests
+    {
+        private readonly NationalMinimumWageService _service;
+        private readonly Mock<ILogger<NationalMinimumWageService>> _mockLogger;
+        
+        public NationalMinimumWageServiceTests()
+        {
+            _mockLogger = new Mock<ILogger<NationalMinimumWageService>>();
+            var mockDataProvider = new Mock<IMinimumWageDataProvider>();
+            mockDataProvider.Setup(x => x.GetMinimumWageData()).ReturnsAsync(GetTestData());
+
+            _service = new NationalMinimumWageService(mockDataProvider.Object, _mockLogger.Object);
+        }
+
+        [Fact]
+        public void ShouldPickCorrectWageRageBasedOnDate()
```

Comment:
😡 Rage

Reply:
Took me a while to work out why @sachpatel was so upset! ShouldPickCorrectWage**Rage**BasedOnDate